app/controllers/runtime/organizations_controller.rb

```diff
@@ -214,6 +228,20 @@ def share_domain(guid, domain_guid)
       [HTTP::MOVED_PERMANENTLY, headers, 'Use DELETE /v2/private_domains/:domain_guid']
     end
 
+    get '/v2/organizations/:guid/isolation_segments', :list_isolation_segments
+    def list_isolation_segments(guid)
+      org = find_guid_and_validate_access(:read, guid)
+
+      dataset = IsolationSegmentModel.dataset.where(organizations: org)
+      message = IsolationSegmentsListMessage.from_params(@params)
+
+      [HTTP::OK, MultiJson.dump(Presenters::V3::PaginatedListPresenter.new(
```

Comment:
@Gerg Thanks for checking with Nick on this.  If you can let me know when you've finished merging, I'll see about a follow-on PR to implement a v2 presenter for Isolation Segments.

-Sandy